### PR TITLE
feature: toggle on payments and scan

### PR DESCRIPTION
**Pull request by akomisarek**
### Change description ###

Toggles on payments and scan on prod.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/WorkBasketInputFields/WorkBasketInputFields.json [hunk 1]
```diff
@@ -33,5 +33,12 @@
     "CaseFieldID": "dateSubmitted",
     "Label": "Date submitted",
     "DisplayOrder": 5
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "evidenceHandled",
+    "Label": "Supplementary evidence handled",
+    "DisplayOrder": 5
```
Comment (tomaszpowroznik) [inline ccd-definition/WorkBasketInputFields/WorkBasketInputFields.json]: should be 6 i guess

File: ccd-definition/CaseEventToComplexTypes/upload-c2/C2DocumentBundle-PREPARE_FOR_HEARING-nonprod.json [hunk 2]
```diff
@@ -1,43 +1,2 @@
...
-    "ID": "C2DocumentBundleComplexType",
-    "CaseEventID": "uploadC2-PREPARE_FOR_HEARING",
-    "CaseFieldID": "temporaryC2Document",
-    "ListElementCode": "fileReference",
-    "EventElementLabel": "File reference",
-    "FieldDisplayOrder": 4,
-    "DisplayContext": "OPTIONAL"
-  }
-]
+[]
```
Comment (tomaszpowroznik) [inline ccd-definition/CaseEventToComplexTypes/upload-c2/C2DocumentBundle-PREPARE_FOR_HEARING-nonprod.json]: file could be removed

File: ccd-definition/WorkBasketInputFields/WorkBasketInputFields.json (hunk 1 above)
Comment (akomisarek) [reply ccd-definition/WorkBasketInputFields/WorkBasketInputFields.json]: Done

File: ccd-definition/CaseEventToComplexTypes/upload-c2/C2DocumentBundle-PREPARE_FOR_HEARING-nonprod.json (hunk 2 above)
Comment (akomisarek) [reply ccd-definition/CaseEventToComplexTypes/upload-c2/C2DocumentBundle-PREPARE_FOR_HEARING-nonprod.json]: Updated with Piotr's PR

**Pull request merged**